INSTALL.md

```diff
@@ -1,5 +1,35 @@
+## Operating system support
+
+Dangerzone can run on a varieties of Operating systems, and have automated tests for a subset of them.
+This section explains what is the support you should expect from us.
+
+You can find general support information in this table, and more details in the following sections.
+
+(Unless specified, the architecture is AMD64)
+
+| Distribution | Supported versions        | Automated tests        | Manual QA         |
+| ------------ | ------------------------- | ---------------------- | ----------------- |
+| Windows      | 3 last releases           | Last release only      | Last version only |
+| macOS intel  | 3 last releases           | Last release only      | Last release only |
+| macOS ARM    | 3 last releases           | Last release only      | Last release only |
+| Ubuntu       | Follow upstream support ✰ | 🗹                      | 🗹                |
+| Debian       | Current stable, Oldstable and LTS releases | 🗹     | 🗹                |
+| Fedora       | Follow upstream support   | 🗹                      | 🗹                |
+| Qubes OS     | [Beta support](https://github.com/freedomofpress/dangerzone/issues/413) ✢ | 🗷 | Latest Fedora template |
+| Tails        | Only the latest version   | 🗷 | Last release only |
+
+Notes:
+
+✰ Support for Ubuntu Focal [was dropped](https://github.com/freedomofpress/dangerzone/issues/1018)
+
+✢ Qubes OS support is done via the use of other OSes (e.g. Fedora or Debian) for the VMs, so the versions we support are tied to them.
```

Comment:
```suggestion
✢ Qubes OS support assumes the use of a Fedora template. The supported versions follow our general support for Fedora.
```

Reply:
Thanks! Updated in c0a4f388b36669d0fdbe3726b2f2cc47e7cdca28